docs/general/resources.md

```diff
@@ -53,15 +58,21 @@ These products use Slate, and can give you an idea of what's possible:
 
 These pre-packaged editors are built on top of Slate, and can be helpful to see how you might structure your code:
 
-- [Accord Project Markdown Editor](https://github.com/accordproject/web-components) is a WYSIWYG editor for [CommonMark](https://commonmark.org/).
+- [Accord Project Markdown Editor](https://github.com/accordproject/web-components) is a WYSIWYG editor
+  for [CommonMark](https://commonmark.org/).
 - [Canner Editor](https://github.com/Canner/canner-slate-editor) is a rich text editor.
-- [Chatterslate](https://github.com/chatterbugapp/chatterslate) helps teach language grammar and more at [Chatterbug](https://chatterbug.com).
+- [Chatterslate](https://github.com/chatterbugapp/chatterslate) helps teach language grammar and more
+  at [Chatterbug](https://chatterbug.com).
 - [CoCalc](https://github.com/sagemathinc/cocalc/) Collaborative Calculation editor in the Cloud
-- [French Press Editor](https://github.com/roast-cms/french-press-editor) is a customizeable editor with offline support.
+- [French Press Editor](https://github.com/roast-cms/french-press-editor) is a customizeable editor with offline
+  support.
 - [Nossas Editor](http://slate-editor.bonde.org/) is a drop-in WYSIWYG editor.
-- [React Force Slate Editor](https://github.com/nareshbhatia/react-force/tree/master/packages/slate-editor) is a light-weight medium-style editor with no editor chrome.
-- [React Page](https://github.com/react-page/react-page) is a self-contained, customizable inline WYSIWYG editor library.
-- [Slate Plugins Next](https://github.com/zbeyens/slate-plugins-next) provides an editor with configurable and extendable plugins.
+- [React Force Slate Editor](https://github.com/nareshbhatia/react-force/tree/master/packages/slate-editor) is a
+  light-weight medium-style editor with no editor chrome.
+- [React Page](https://github.com/react-page/react-page) is a self-contained, customizable inline WYSIWYG editor
+  library.
+- [Slate Plugins Next](https://github.com/zbeyens/slate-plugins-next) provides an editor with configurable and
```

Comment:
Not really relevant to your change, but might as well since I noticed how old this link is!

```suggestion
- [Plate (Plugins for Slate)](https://github.com/udecode/plate) provides an editor with configurable and
```

Reply:
Got it!